spec/event_sourcery/postgres/optimised_event_poll_waiter_spec.rb

```diff
@@ -81,4 +83,17 @@ def notify_event_ids(*ids)
       end
     end
   end
+
+  def quiet_thread_report_on_exception(&block)
```

Comment:
Perhaps move this method definition into `spec/support/`.

Reply:
Ah, sorry. I merged this pretty quickly. :/